src/generic/datavgen.cpp

```diff
@@ -3377,47 +3422,68 @@ int wxDataViewMainWindow::GetLineStart( unsigned int row ) const
     }
 }
 
-int wxDataViewMainWindow::GetLineAt( unsigned int y ) const
+int wxDataViewMainWindow::GetLineAt( int y ) const
 {
+    if (y < 0) return -1;
```

Comment:
Same comment as for `RefreshRow()` above: IMO we should assert here, not return silently.

Also, unrelated, but please don't write `if` statements on a single line. If nothing else, this makes it impossible to put a breakpoint on `return` statement in most (all?) debuggers and this would be a very useful thing to do in this particular case in order to find out when this happens.

Reply:
removed it. it is probably not necessary anymore